### PR TITLE
A models function that checks if there is an alert for the user + Test

### PR DESCRIPTION
**Pull request by Varditli**
#81

### CODE REVIEW

File: user_notification/tests.py [hunk 1]
```diff
@@ -24,3 +26,34 @@ def test_exception(n, x):
...
+@pytest.mark.django_db
+def test_check_if_alert_exist(n, expected):
+    insertDataToDB()
+    assert str(JobAlert.check_if_alert_exist(n)) == expected
+    deleteAllobjects()
+
+
+def deleteAllobjects():
+    JobAlert.objects.all().delete()
+    UserAccount.objects.all().delete()
```
Comment (mayasa3) [inline user_notification/tests.py]: UserAccount line isn't needed, deleting a user automatically deletes the connected UserAccount

File: user_notification/tests.py [hunk 2]
```diff
@@ -1,6 +1,8 @@
 from user_notification.models import JobAlert
 import pytest
 from datetime import datetime
+from accounts.models import UserAccount
```
Comment (mayasa3) [inline user_notification/tests.py]: Remove this import too

File: user_notification/tests.py (hunk 1 above)
Comment (mayasa3) [inline user_notification/tests.py]: Remove this line, deleting the User will automatically delete UserAcount

File: user_notification/tests.py (hunk 2 above)
Comment (mayasa3) [inline user_notification/tests.py]: Import not needed unless you need to reference user details stored in UserAccount(birth date, gender or contact number)

**Review by mayasa3 — APPROVED** [review]

File: user_notification/tests.py [hunk 3]
```diff
@@ -1,7 +1,6 @@
 from user_notification.models import JobAlert
 import pytest
 from datetime import datetime
-from accounts.models import UserAccount
```
Comment (bardielle) [inline user_notification/tests.py]: please squash that commit with the previous one

File: user_notification/tests.py (hunk 2 above)
Comment (Varditli) [reply user_notification/tests.py]: I edited this in the next PR, and deleted this import

File: user_notification/tests.py (hunk 1 above)
Comment (Varditli) [reply user_notification/tests.py]: I edited this function in the next PR, thanks!

File: user_notification/tests.py (hunk 3 above)
Comment (Varditli) [reply user_notification/tests.py]: I squashed them, thanks

Comment (bardielle) [reply user_notification/tests.py]: It seems that you still have 3 commits (instead of 2)

Comment (bardielle) [reply user_notification/tests.py]: @Varditli ?

**Comment by bardielle** [comment]
The tests are failing...  please fix it

**Comment by guy9050** [comment]
@Varditli is this PR ready for merge? I see the tests passed and you have 2 commits

File: user_notification/tests.py (hunk 3 above)
Comment (Varditli) [reply user_notification/tests.py]: I squashed them

**Comment by Varditli** [comment]
yes it is ready @guy9050

**Review by bardielle — APPROVED** [review]
/lgtm

**Pull request merged**